src/index.mako.html

```diff
@@ -217,6 +217,7 @@
   % if device == 'mobile':
         <div ga-offline-bt></div>
   % endif
+        <div ga-tilt3d ng-if="globals.dev3d"></div>
```

Comment:
Ok, It may be quite the same here. Personally I tend to use `ng-if` only when strictly necessary.